src/crimson/os/seastore/cached_extent.h

```diff
@@ -574,12 +574,19 @@ class CachedExtent
     return state == extent_state_t::INITIAL_WRITE_PENDING;
   }
 
-  /// Returns true if extent is clean (does not have deltas on disk)
-  bool is_clean() const {
+  /// Returns iff extent has deltas on disk or pending
+  bool has_delta() const {
```

Comment:
IIUC, `has_delta` is looser than `is_dirty`, which may weaken the asserts in `SeaStore`, is this worth it?

Reply:
I think there is no change except for a better name to reflect this fact, actually.

I also plan to improve this after hiding the pending states.

Reply:
Ah, yes, I've been thinking `is_dirty()` is `state == DIRTY`.